accelerator/core/core.h

```diff
@@ -215,6 +217,23 @@ status_t ta_send_bundle(const iota_config_t* const iconf, const iota_client_serv
 status_t ta_get_bundles_by_addr(const iota_client_service_t* const service, tryte_t const* const addr,
                                 bundle_array_t* bundle_array);
 
+/**
+ * @brief Get current connection status. The status will be responded with return value.
+ *
+ * We would check out the connection status with IRI core API getNodeInfo. At the first step, we would check whether
+ * tangle-accelerator can connect to the IRI host which is assigned in iota_client_service_t object. If the
+ * tangl-accelerator connects to the IRI host, the next step we are going to check out whether the connected IRI has
+ * synchronized to the latest milestone. The mentioned two errors above would trigger tangle-acclerator connect to
+ * another IRI host on IRI priority host list.
```

Comment:
Is the switching host process executed by TA, or other server?

Reply:
for the first stage it will be executed by TA